velox/exec/Unnest.h

```diff
@@ -106,6 +107,13 @@ class Unnest : public Operator {
       const RowRange& rowRange,
       std::vector<VectorPtr>& outputs);
 
+  // Calculate the max number of elements of each row after unnested.
+  // @param numRows The number of input rows.
+  // @param isOuter Whether we should generate a null element.
+  // if the array/map is null or empty then null is produced.
```

Comment:
> If true, emit null data for empty array/map or array/map with null elements.

The description seems different with that in PlanNode.h. One is `array/map is null`, and another is `array/map with null elements`. Would you confirm?